### PR TITLE
cds bind supports custom credentials

### PR DESCRIPTION
**Pull request by LotharBender**

### CODE REVIEW

File: advanced/hybrid-testing.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ uacp: Used as link target from Help Portal at https://help.sap.com/products/BTP/
 
 CAP enables you to run and test your CAP application using a local SQLite database and mocks to a large extent. However, you might want to test with actual cloud services at some point.
 
-**Hybrid testing** capabilities help you stay in a local development environment and avoid long turnaround times of cloud deployments, by selectively connecting to services in the cloud.
+**Hybrid testing** capabilities help you stay in a local development environment and avoid long turnaround times of cloud deployments, by selectively connecting to services in the cloud. With hybrid testing you can also overwrite dedicated service credential values.
```
Comment (renejeglinsky) [inline advanced/hybrid-testing.md:14]: Why would we mention this explicitly at the beginning of the guide? Is it more important than other `cds bind` usage scenarios? We haven't even mentioned how credentials are handled. Wouldn't this be a prerequisite to understand this sentence?

File: advanced/hybrid-testing.md [hunk 2]
```diff
@@ -363,6 +366,60 @@ cds bind -2 my-hana,my-destination,my-xsuaa
...
+
+```sh
+cds watch --profile hybrid
+```
+
+Example output:
+
+```js
+{
+  url: 'jdbc:sap://BDB9AC0F20CB46B494E6742047C4F99A.hana.eu10.hanacloud.ondemand.com:443?encrypt=true&validateCertificate=true&currentschema=BDB9AC0F20CB46B494E6742047C4F99A',
```
Comment (renejeglinsky) [inline advanced/hybrid-testing.md:415]: Is this a URL we should obscure?

File: advanced/hybrid-testing.md [hunk 3]
```diff
@@ -363,6 +366,60 @@ cds bind -2 my-hana,my-destination,my-xsuaa
 This shortcut is only possible if you don't need to provide a `service` or a `kind`.
 :::
 
+### Overwrite BTP Service Credentials
+
+Some hybrid test scenarios might require to overwrite dedicated service credential values. For example, when connecting to an SAP S/4HANA On-Premise system you may need to overwrite _onpremise_proxy_host_ and _onpremise_proxy_port_ values.
```
Comment (renejeglinsky) [inline advanced/hybrid-testing.md:371]: What is the scenario? I'd like to understand how BTP service and onPrem system work together here. Thank you :)

Comment (LotharBender) [reply advanced/hybrid-testing.md:372]: Used ssh tunnel scenario instead which is consistent with `cds deploy --tunnel-address`
But I have to admit that I do not know any further details regarding the concrete use case for this...

File: advanced/hybrid-testing.md (hunk 2 above)
Comment (LotharBender) [reply advanced/hybrid-testing.md:414]: copied from above example - in that case it needs to be obscured for both..

File: advanced/hybrid-testing.md (hunk 1 above)
Comment (LotharBender) [reply advanced/hybrid-testing.md]: Already deleted this. It is sufficient to have `Overwrite Service Credentials` listed in the table of contents.

**Review by chgeo — APPROVED** [review]

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**